package.json

```diff
@@ -9,6 +9,7 @@
   "main": "./lib/api.js",
   "scripts": {
     "test": "node Makefile.js test",
+    "mocha": "./node_modules/.bin/mocha",
```

Comment:
My apologies, I meant to say you could get rid of the path prefix due to how `npm run` works.

```
"test:cli": "mocha"
```

Reply:
Oh sorry, I see

Reply:
No need to apologize- this was my fault. :smile: Thanks for your patience!